### ISSUE TITLE
Out-of-date usage string in the python-urllib3 stub

### ISSUE THREAD

**Issue by jviide** [ouspg/trytls#112]
The current `stubs/python-urllib3/run.py` version prints an out-of-date usage string:

``` sh
$ python stubs/python-urllib3/run.py 
Usage: stubs/python-urllib3/run.py <URL> <PORT> [CA_FILE]
```

Notice the `<URL>` part, which probably should be `<HOST>` or somesuch.

**Comment by jviide** [comment]
Commit 93ac0f2 fixes the python3-urllib stub instead of python-urllib3. This issue was actually fixed in commit a7eda41.

**Comment by Mamietti** [comment]
I was just testing whether someone would notice, naturally.
